README.md

```diff
@@ -115,7 +117,7 @@ start();
 
 :warning: It's important to note that you must also use the [`enableA11yAudit`](#force-running-audits) query parameter in order to force audits. This setting is required in addition to any invocation strategy you provide.
 
-By default, audits will be run on `visit`, `click`, `doubleClick`, and `tap`. To add additional helpers to hook into, specify them by name in the `options.helpers` argument. Note that this option specifies the *complete* set of helpers to hook into; to include the defaults you must import them and splat them into the array as shown below.
+By default, audits will be run on `visit`, `click`, `doubleClick`, and `tap`. To add additional helpers to hook into, specify them by name in the `options.helpers` argument. Note that this option specifies the _complete_ set of helpers to hook into; to include the defaults you must import them and splat them into the array as shown below.
```

Comment:
Most of these changes were unintentional but ran since I have the prettier plugin enabled in VSCode. Should we leave the changes in and figure out how to have prettier check `.md` files in CI in a different PR?

Reply:
Yes, I think it's fine to leave them in, can you create an issue to track this though?

Reply:
#510